lab-tracey-radcliffe/lib/readFiles.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const EE = require('events');
+
+const readFiles = function(files, callback) {
+  var array = [];
```

Comment:
'array' is a reserved keyword in JavaScript, so it would be better to rename your variable to something like 'arr'.